README.md

```diff
@@ -1,10 +1,33 @@
 ![Unleash_GitHub_Community_Content](https://user-images.githubusercontent.com/18158596/210858584-7b8e0e7f-dcf8-4795-a868-0d3c9dca2edb.png)
 
-# Community Content Program ✍️
+<div align="center">
+  <h1>Community Content Program ✍️</h1>
+  <p>Welcome to the Unleash Community Content Program! We are excited to have knowledgeable users in the community contribute to Unleash in their own way by sharing their experience on DevOps, Software Delivery, and Unleash.</p>
+  <a href="https://www.getunleash.io/">Website</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://www.docs.getunleash.io/">Docs</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://github.com/Unleash/unleash/">GitHub</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://www.getunleash.io/blog">Blog</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://slack.unleash.run/">Slack</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://twitter.com/prisma">Twitter</a>
+    <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+    <a href="https://linkedin.com/company/getunleash">LinkedIn</a>
+  <hr />
+</div>
```

Comment:
Why is this written out in HTML instead of markdown? Is there something you couldn't do with markdown here?

For maintainability, I'd suggest leaving it as markdown, but if you've got a specific need, then that's fine too 💁🏼